crates/ere-dockerized/src/lib.rs

```diff
@@ -215,6 +216,10 @@ impl EreDockerizedCompiler {
             mount_directory: mount_directory.as_ref().to_path_buf(),
         }
     }
+
+    pub fn zkvm(&self) -> ErezkVM {
+        self.zkvm
+    }
```

Comment:
Adding this so `zkevm-benchmark-workload` can match the `zkvm` and decide to deserialize the output and check (`sp1` and `risc0`), or hash the expected output and check (`openvm` and `zisk`).